kale/pipeline/fewshot_trainer.py

```diff
@@ -0,0 +1,124 @@
+# =============================================================================
+# Author: Wenrui Fan, winslow.fan@outlook.com
+# =============================================================================
+
+"""ProtoNet trainer (pipelines)
```

Comment:
Update this and state the purpose of this API as you said above?

Reply:
Done